Tests/ContentKeyPeriodCrudTests.cs

```diff
@@ -0,0 +1,228 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml.Schema;
+using Xunit;
+
+namespace Axinom.Cpix.Tests
+{
+	public sealed class ContentKeyPeriodCrudTests
+	{
+		[Fact]
+		public void AddContentKeyPeriod_WithLoadedEmptyDocument_Succeeds()
+		{
+			var document = new CpixDocument();
+			document = TestHelpers.Reload(document);
+
+			document.ContentKeyPeriods.Add(new ContentKeyPeriod { Index = 1 });
+			document = TestHelpers.Reload(document);
+
+			Assert.Single(document.ContentKeyPeriods);
+		}
+
+		[Fact]
+		public void AddContentKeyPeriod_WithLoadedDocumentWithExistingContentKeyPeriod_Succeeds()
+		{
+			var document = new CpixDocument();
+			document.ContentKeyPeriods.Add(new ContentKeyPeriod { Index = 1 });
+			document = TestHelpers.Reload(document);
+
+			document.ContentKeyPeriods.Add(new ContentKeyPeriod { Index = 2 });
+			document = TestHelpers.Reload(document);
+
+			Assert.Equal(2, document.ContentKeyPeriods.Count);
+		}
+
+		[Fact]
+		public void AddContentKeyPeriod_WithVariousValidData_Succeeds()
+		{
+			var document = new CpixDocument();
+
+			Assert.Null(Record.Exception(() => document.ContentKeyPeriods.Add(new ContentKeyPeriod
```

Comment:
This seems to be a weird way to do this? Don't we usually use Assert.DoesNotThrow?

Reply:
Oh, is this Xunit? Perhaps that's why.